### ISSUE TITLE
Backbone pretrained part

### ISSUE THREAD

**Issue by rachitagarwal2603** [ZrrSkywalker/MonoDETR#7]
In your pretrained model resnet-50 backbone part, I can see there are some extra Batch Normalisation layers like bn.weight, bn.bias, bn.running_var, bn.running_mean for each backbone layer...

While loading model, these layers are ignored. I wanted  to understand the significance of these layers. Please let me know.
